dpdk/vr_dpdk_virtio.c

```diff
@@ -60,6 +60,29 @@ vr_dpdk_virtio_ntxqs(struct vr_interface *vif)
 }
 
 /*
+ * dpdk_virtio_rx_queue_release - releases a virtio RX queue.
+ *
+ * Returns nothing.
+ */
+static void
+dpdk_virtio_rx_queue_release(unsigned lcore_id, struct vr_interface *vif)
+{
+    struct vr_dpdk_lcore *lcore = vr_dpdk.lcores[lcore_id];
+    struct vr_dpdk_queue *rx_queue = &lcore->lcore_rx_queues[vif->vif_idx];
+    struct vr_dpdk_queue_params *rx_queue_params
+                        = &lcore->lcore_rx_queue_params[vif->vif_idx];
+
+    /* remove the ring from the list of rings to push */
+    dpdk_ring_to_push_remove(rx_queue_params->qp_ring.host_lcore_id,
+            rx_queue_params->qp_ring.ring_p);
```

Comment:
The ring itself needs to be freed? (it was allocated with rte_ring_create)? Also, would be good to zero out vr_dpdk_virtio_rxqs[vif_idx][queue_id].